src/api/protectedApiClient.ts

```diff
@@ -40,6 +44,16 @@ const deactivateAccount = (): Promise<void> => {
     .catch((err) => err);
 };
 
+const getContactInfo = (): Promise<ContactInfo> => {
+  return AppAxiosInstance.get(ProtectedApiClientRoutes.CONTACT_INFO).then(
+    (res) => res.data,
+  );
+};
+
+const setContactInfo = (request: ContactInfo): Promise<void> => {
+  return AppAxiosInstance.put(ProtectedApiClientRoutes.CONTACT_INFO, request);
```

Comment:
I actually like this. No need for a then/catch when its returning a `Promise<void>`. Can you quickly fix that for the others ones too(`makePFRequest`, `changePassword `, `deactivateAccount`, etc)?